Software-Engineering/Overall-Ratings.md

```diff
@@ -21,13 +23,17 @@ Use these tests to check if someone is ready for a promotion to Maturing at the
 
 There is no hard and fast rule for a maximum duration that someone can be in a Maturing band. The important thing is that the manager is confident about the person's growth trajectory toward Performing. If this confidence is lost, the manager and person should discuss moving back to Exceeding in the previous level.
 
+To progress from the Maturing band to the Performing band at a particular level, someone will typically need to have demonstrated a track record of performing at that level autonomously and without requiring significant support from their manager to do so.
+
 ### Performing
 
-The Performing band should be considered the default, and is for everyone who typifies what it means to be a great contributor at that level at Octopus.
+The Performing band should be considered the default and is for everyone who typifies what it means to be a great contributor at that level at Octopus. 
 
 ### Exceeding
 
-The Exceeding band is a tool that managers can use when they need to recognize value contributed significantly _above and beyond_ what is expected at their level. This should usually only apply to:
+The Exceeding band is a tool that managers can use when they need to recognize value contributed significantly _above and beyond_ what is expected at their level. _Above and beyond_ is typically demonstrated by exhibiting behaviours from the level above or by operating with an impact radius larger than that expected in their current role while continuing to perform the requirements of their current level effectively.
```

Comment:
I struggle with this one - to me, this reads as they are performing between their current role and the next. But the salary is the same, so they should be contributing similar value based on our "printer test" principle. 

@mattocto happy for you to clear this one up for me :)

Reply:
@gian-lorenzetto I think I can help :) 

By going above and beyond their current role, they're performing between the _performing band for their current role_ and the _performing band for the next role_. This could be done either as an Exceeding in their current role or a Maturing in the next one for the same salary, thereby passing the printer test, even when the level is different.

The reasons for using the Exceeding band in this case, rather than the Maturing band are the ones listed below (unchanged from the previous version).

Does that help? 🤞

Reply:
Just to replay that to make sure I understand - the Exceeding band is more of an exception, and the expectation is that the IC is contributing very closely to the Maturing of the next band, but there are specific circumstances preventing that (they don't want the responsibilities or we've asked someone to take on responsibilities of the next role and want to recognise that).

The confusion I've seen is whether the Exceeding band is a stepping stone to the next role. That is, because they have an "exceeding" rating in the Performing role, there is an expectation to go to the "Exceeding" band, even though they still have gaps to close. 

I think this clarifies it for me. 🙇

Reply:
Ideally, the terminology for performance reviews "Exceeds Expectations" would be completely decoupled from the levels/bands in Octopus People "Level `X` Exceeding", but that's beyond the scope of this piece of work.

It sounds like we're aligned on what the Exceeding band is for and when it might be used, and that's as good as we can reasonably aim for right now. :)